internal/app/export/service.go

```diff
@@ -47,16 +46,16 @@ func New(nc *nats.Conn, mqtt mqtt.Client, c config.Config, logger log.Logger) Se
 // and makes routes according to the configuration file.
 // Routes export messages to mqtt.
 func (e *exporter) Start(queue string) {
-	var route routes.Route
+	var route routes.RouteIF
```

Comment:
You could use `route := routes.RouteIF{}` here

Reply:
i think it doesnt work, RouteIF is interface